bin/spice/cmd/dataset.go

```diff
@@ -200,16 +200,16 @@ spice dataset configure
 
 		var datasetReferenced bool
 		for _, dataset := range spicePod.Datasets {
-			if dataset.Ref == dirPath {
+			if dataset.Reference != (spec.Reference{}) && dataset.Reference.Ref == dirPath {
```

Comment:
this line could be a method in dataset?

Reply:
I had naively thought Go would complain about `Reference` being an empty value, thanks for the pointer!